src/py42/services/alertrules.py

```diff
@@ -0,0 +1,136 @@
+from py42.services import BaseService
```

Comment:
im not sure why this whole file is green, much like the others it just has naming changes / accomodations for not needing microserviceclientfactory anymore

Reply:
> im not sure why this whole file is green, much like the others it just has naming changes / accomodations for not needing microserviceclientfactory anymore

it has moved to a new directory `services` which din't exist earlier.

Reply:
many other files were moved to that directory (everything that used to be in `clients`, see my renaming message in the PR description), and they still had the diff displayed appropriately.

Reply:
That happens sometimes, I think it might have to do with how you moved it.